.github/workflows/go-check.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         if: ${{ success() || failure() }} # run this step even if the previous one failed
         uses: protocol/multiple-go-modules@v1.2
         with:
-          run: go vet ./...
+          run: go vet -composites=false ./...
```

Comment:
The next deployment would overwrite this.

Reply:
Is there a way to not have this problem?